pages/upgrade/graylog-2.4.rst

```diff
@@ -4,6 +4,9 @@ Upgrading to Graylog 2.4.x
 
 .. _upgrade-from-23-to-24:
 
+You can just upgrade from Graylog 2.3.x to Graylog 2.4.x without the need to change the configuration of your Graylog server. 
```

Comment:
Remove the "just".

➡️  https://www.cherryleaf.com/blog/2018/01/dont-say-simply/